src/shipit_static_analysis/tests/mocks/config.yaml

```diff
@@ -6,4 +6,4 @@ clang_checkers:
  - name: clang-analyzer-security.*
    publish: !!bool no
 
-third_party: tools/rewriting/ThirdPartyPaths.txt
+third_party: 3rdparty.txt
```

Comment:
Wait, for real?

Reply:
That's a mock config file, with simpler paths. Don't worry it's only used by unit tests through `pytest`. The bot downloads the real file from mozilla-central.